### ISSUE TITLE
[suggest]TWO-WAY control

### ISSUE THREAD

**Issue by unbadfish** [Robbendebiene/Gesturefy#477]
I think that this add-on should be able to work in both "stpuid" way (up/down/left/right) OR "intellgent" way, for I always draw the Gesturefy in a "UNstandard" way:cry:
**Maybe U can add the old "stpuid" way back?** I(and maybe lots of others) think that the old way is more useful
There coule be a `RADIO Button` for users to choose how Gesturefy works--in "stpuid" way or "intellgent" way--and user could have TWO series of settings.(and both the versions setting file .json shoule be able to upload)
Maybeit's not a hard work, as version 2.X.X works very well.

**Comment by lleaff** [comment]
It would probably help if you described more precisely what you don't like about the new system. Is it that the new gesture detection allows less room for error?

I'm not going to answer for the maintainer but I doubt having the two systems in parallel is a good idea for maintainability. In the meantime if you want an add-on with the simpler system there's [Foxy Gestures](https://addons.mozilla.org/en-US/firefox/addon/foxy-gestures/) which also works well.

**Comment by unbadfish** [comment]
>  Is it that the new gesture detection allows less room for error?

Yes. 
For example, I draw 2 kinds of `down->right->up`,one like"U", one like "\_/",and I found it hard to define them

> I doubt having the two systems in parallel is a good idea for maintainability.

The RADIO Button provides a "A or B"choice, so the user must choose--the "stupid" one or the "smart" one
So only one system can be used at one time, and them may update separately(Maybe)

**Comment by Itchiii** [comment]
@lleaff is right, in the background (of programming) it does not remain with A or B as radio buttons

Duplicate of #486

**Comment by Itchiii** [comment]
Duplicate of #486
